packages/engine/src/worker/runner/python/sender.rs

```diff
@@ -118,6 +118,8 @@ impl NngSender {
 
 impl Drop for NngSender {
     fn drop(&mut self) {
+        // Python has to close the socket on its side for the "-topy*" file to be deleted.
+        let _ = self.send(None, &InboundToRunnerMsgPayload::TerminateRunner, &None);
         // TODO: Check whether nng already does this when a socket is dropped
```

Comment:
Is this still up-to-date?

Reply:
Nng will close the socket when all handle to it are dropped.
I guess we only have a single handle per `NngSender` so it seems to be already taken care off by nng.

Reply:
I removed it and the whole `Drop` too.